tests/sandbox/sandbox.sh

```diff
@@ -20,12 +20,16 @@ source "${VENV_DIR}"/bin/activate
 
 if [ -f "${VERSION_FILE}" ]; then
     # install tree
+    BINARY_DIR=${PATH_HERE}
+    LIBRARY_DIR="${PATH_HERE}"/../lib
     START_NETWORK_SCRIPT="${PATH_HERE}"/start_network.py
     VERSION=$(<"${VERSION_FILE}")
     pip install --disable-pip-version-check -q -U ccf=="$VERSION"
     pip install --disable-pip-version-check -q -U -r "${PATH_HERE}"/requirements.txt
 else
     # source tree
+    BINARY_DIR="${PATH_HERE}"/../../build
```

Comment:
```suggestion
    BINARY_DIR="."
```

Minor, but if I'm in a folder called `build.debug` or `build.san`, I'd like to be able to use `sandbox.sh` with my local binaries. If I'm outside of any `build` directory, I _can_ still use `sandbox.sh`, but I need to specify `--binary-dir` explicitly to say which `cchost` I want to use.